src/Files/ViewModels/Dialogs/FilesystemOperationDialogViewModel.cs

```diff
@@ -106,16 +108,28 @@ public bool MustResolveConflicts
 
         public ICommand ApplyToAllCommand { get; private set; }
 
+        public ICommand ClosingCommand { get; private set; }
+
+        private CancellationTokenSource ClosingCts { get; }
+
         public FilesystemOperationDialogViewModel()
         {
+            ClosingCts = new CancellationTokenSource();
```

Comment:
One more thing. I think the cts needs to be disposed

Reply:
Thankfully no, my understanding is that calling Dispose is only required if:
- you use CancelAfter which sets up a timer
- you access the WaitHandle member
As you can see in ItemsViewModel we never Dispose of cancellation tokens as it tends to create issues when you try to cancel a bunch of threads and then the token is Disposed before all of them have been stopped.

Reply:
Nice! Didn't know that